autogpt/config/config.py

```diff
@@ -110,6 +110,7 @@ def __init__(self) -> None:
         # Selenium browser settings
         self.selenium_web_browser = os.getenv("USE_WEB_BROWSER", "chrome")
         self.selenium_headless = os.getenv("HEADLESS_BROWSER", "True") == "True"
+        self.selenium_proxy_server = os.getenv("PROXY_SERVER", None)
```

Comment:
Documentation with examples are missing from the .env.template file for this variable. Also it might be better to use a variable name which refers to the browser, since I suppose other requests will be not proxied, so something like: `PROXY_SERVER_FOR_BROWSER`

Reply:
Will tweak it.